### PR TITLE
Add options and write EV code

### PR DESCRIPTION
**Pull request by vtnate**
Also includes writing new options/arguments in `measure.rb`, which should be closely examined for errors due to my inexperience with that task.

### CODE REVIEW

**Comment by vtnate** [comment]
Commit `3d49a2c` needs to be examined extra closely. Cargo-cult programming at its best.

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -1686,12 +1689,23 @@ Misc Hot Tub Spa	None	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=
...
 Misc Hot Tub Spa	Gas	ResidentialMiscLargeUncommonLoads	has_hot_tub_heater_elec=false	has_hot_tub_heater_gas=true	has_hot_tub_pump=true	hot_tub_heater_elec_annual_energy=1027.3	hot_tub_heater_elec_mult=1.0	hot_tub_heater_gas_annual_energy=81	hot_tub_heater_gas_mult=1.0	hot_tub_pump_annual_energy=1014.1	hot_tub_pump_mult=1.0	hot_tub_scale_energy=true	"hot_tub_weekday_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_weekend_sch=0.024, 0.029, 0.024, 0.029, 0.047, 0.067, 0.057, 0.024, 0.024, 0.019, 0.015, 0.014, 0.014, 0.014, 0.024, 0.058, 0.126, 0.122, 0.068, 0.061, 0.051, 0.043, 0.024, 0.024"	"hot_tub_monthly_sch=0.921, 0.928, 0.921, 0.915, 0.921, 1.160, 1.158, 1.158, 1.160, 0.921, 0.915, 0.921"																																			
+Misc Hot Tub Spa Pump
```
Comment (joseph-robertson) [inline resources/options_lookup.tsv]: Are these two lines supposed to be here?

File: project_singlefamilydetached/housing_characteristics/Misc Pool Pump.tsv [hunk 2]
```diff
@@ -1,2 +1,13 @@
-Option=1.0 HP Pump
-1
+Dependency=Misc Pool	Option=1.0 HP Pump
+None	0
+National Average	1
+Electric	1
```
Comment (joseph-robertson) [inline project_singlefamilydetached/housing_characteristics/Misc Pool Pump.tsv]: All your dependencies here should be listed as options in Misc Pool.tsv.

File: project_multifamily_beta/housing_characteristics/Misc Pool Pump.tsv [hunk 3]
```diff
@@ -1,2 +1,13 @@
-Option=1.0 HP Pump
-1
+Dependency=Misc Pool	Option=1.0 HP Pump
+None	0
+National Average	1
+Electric	1
```
Comment (joseph-robertson) [inline project_multifamily_beta/housing_characteristics/Misc Pool Pump.tsv]: All your dependencies here should be listed as options in Misc Pool.tsv.

File: project_testing/housing_characteristics/Misc Pool Pump.tsv [hunk 4]
```diff
@@ -1,2 +1,13 @@
-Option=0.75 HP Pump	Option=1.0 HP Pump
-0.5	0.5
+Dependency=Misc Pool	Option=0.75 HP Pump	Option=1.0 HP Pump	Option=None
+None	0	0	1
+National Average	0.5	0.5	0
+Electric	0.5	0.5	0
```
Comment (joseph-robertson) [inline project_testing/housing_characteristics/Misc Pool Pump.tsv]: All your dependencies here should be listed as options in Misc Pool.tsv.

File: resources/options_lookup.tsv (hunk 1 above)
Comment (vtnate) [reply resources/options_lookup.tsv]: Gah, nope. Good catch.

**Comment by vtnate** [comment]
Forgot to add a `None` column after my changes to `Misc Pool Pump`. It's on its way.

**Comment by vtnate** [comment]
@joseph-robertson I have confirmed that the results from PAT are what I expect, show up when desired and not when not. You can see these results in the `local_results` folder in my branch. Can you find anything else that should be improved in this PR?

**Comment by vtnate** [comment]
@joseph-robertson Ha. I thought of more tests I want to run, and I found what I think is a maintenance bug in my new code. Can you take a quick look at this last commit to make sure it's reasonable?

**Pull request merged**